src/extensions/query-translator/clients/OpenAi/OpenAiClient.ts

```diff
@@ -117,7 +117,7 @@ export class OpenAiClient extends ModelClient {
     let queryExample = reportExampleQueries[reportType];
     let finalMessage = `${content}. Please use the following query structure as an example for ${reportTypesToDesc[reportType]}:
   ${queryExample} 
-  Remember to respect the schema and remove any unnecessary comments or explanations from your result.`;
+  Remember to respect the schema and remove any unnecessary comments or explanations from your result. Remember that every $ prefixed word is a parameter.`;
```

Comment:
I guess this is only to remind the LLM that we keep parameters in the Query. But we don't actually pass the real values for security reasons